app/src/main/res/xml/pref_blacklist.xml

```diff
@@ -9,6 +9,13 @@
             android:summary="@string/pref_summary_blacklist"
             android:title="@string/blacklist" />
 
+        <com.kabouzeid.appthemehelper.common.prefs.supportv7.ATESwitchPreference
```

Comment:
My understanding that white list takes precedence over black list (the track must be in white list first, then will be excluded if in black list).

If this is correct:
- I'd put the corresponding pref before the black list pref.
- and rename the "white list" to "Limit to start folder" (ie white list doesnt seem to be the right wording).
- optionally, rename "black list" to "Exclude content from folder"

Reply:
> My understanding that white list takes precedence over black list (the track must be in white list first, then will be excluded if in black list).
> 
> If this is correct:
> 
> * I'd put the corresponding pref before the black list pref.
> * and rename the "white list" to "Limit to start folder" (ie white list doesnt seem to be the right wording).
> * optionally, rename "black list" to "Exclude content from folder"

Yes, this is the idea. The actual meaning is "Limit to start folder". But blacklist and whitelist name seems more straight-forward. If you think it's better to change, please comment and I'll change.

Reply:
It would make more sense to me to do those changes.
It's really my personal preference though :-)